recipe/meta.yaml

```diff
@@ -14,19 +14,27 @@ source:
     - 0001-no-CC-override.patch
 
 build:
-  number: 1000
-  skip: true  # [win]
+  number: 1001
+  skip: true  # [vc<14]
 
 requirements:
   build:
-    - {{ compiler("c") }}
+    - {{ compiler("c") }}  # [not win]
+    - {{ compiler("m2w64_c") }}  # [win]
   host:
 
 test:
+  requires:  # [win]
+    - {{ compiler("c") }}  # [win]
+  files:  # [win]
+    - test_openlibm_win.c  # [win]
   commands:
     - test -d "${PREFIX}/include/openlibm"                            # [unix]
     - test -f "${PREFIX}/lib/libopenlibm.a"                           # [unix]
     - test -f "${PREFIX}/lib/libopenlibm${SHLIB_EXT}"                 # [unix]
```

Comment:
Should we include some file tests for Windows too?

Reply:
I thought that the compilation test could be sufficient, but there's no problem in including more tests for the files